api/stats/internal.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package nighthawk;
+
+// Internal protos for representing Statistics in a way that allows
+// "lossless" reconstruction. Do not consume / rely on this in any other place
+// than Nighthawk code as this may be subject to change.
+
+message InternalSimpleStatistic {
+  uint64 count = 1;
+  string id = 2;
+  uint64 min = 5;
+  uint64 max = 6;
+  double sum_x = 7;
+  double sum_x_2 = 8;
```

Comment:
Actually can we document both the fields and the messages? While they are internal, documentation would help other developers to understand what they are.

Reply:
In 8ce99fc9 I added a comment, which explains what this file is, and defers to `statistics_impl.h`  for further details.
Would that suffice?

Reply:
Thank you, yes that helps direct the reader to the information they are seeking.